src/Altinn.Notifications.Email/Startup/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Altinn.Notifications.Email.Core;
+using Altinn.Notifications.Email.Integrations.Clients;
+using Altinn.Notifications.Email.Integrations.Configuration;
+using Altinn.Notifications.Email.Integrations.Consumers;
+
+namespace Altinn.Notifications.Email.Startup;
+
+/// <summary>
+/// This class is responsible for holding extension methods for program startup.
+/// </summary>
+public static class ServiceCollectionExtensions
+{
+    /// <summary>
+    /// Add necessary core services and configuration to the service collection.
+    /// </summary>
+    /// <param name="services">The application service collection.</param>
+    /// <param name="config">The application configuration.</param>
+    /// <returns>The given service collection.</returns>
+    public static IServiceCollection AddCoreServices(this IServiceCollection services, IConfiguration config)
+    {
```

Comment:
any reason this is a common extension class rather than having an extension class per project and updating the dependencies there?

Reply:
I'm going back and forth on this myself. This class was originally added to the integration project, but I moved it to simplify dependencies. I'm looking for better arguments for either one if you have some.

Reply:
If we first include all service registrations in an extension method I would say the benefit is that the API project references it once and does not need to be further bothered with following changes to which services are required. If we need to update the api project each time a new service is included we might as well have had the services be registered directly in Program.cs. 

In an app template context, we want to update the project reference/nuget package and be sure end user does not need to make any further changes in the app. I guess we can look at this the same way. With the app template being the API-project.

Reply:
I've now moved the service collection extensions out to the core and integration projects.